package.json

```diff
@@ -125,5 +125,6 @@
       "path": "dist/**/*.js",
       "limit": "800 ms"
     }
-  ]
+  ],
+  "version": "2.2.4"
```

Comment:
Package version automatically sets when releases. There is no reason to put it here?